crates/fluvio/src/lib.rs

```diff
@@ -207,6 +207,35 @@ pub async fn consumer<S: Into<String>>(
     Ok(consumer)
 }
 
+#[cfg(feature = "otel-metrics")]
+pub fn init_open_telemetry() {
+    use std::time::Duration;
+    use opentelemetry::KeyValue;
+    use opentelemetry::sdk::Resource;
+    use opentelemetry::global;
+
+    let resource_attributes = [
+        KeyValue::new("service.name", "client"),
+        KeyValue::new("service.namespace", "fluvio"),
+    ];
```

Comment:
This might need an input parameter to add some way to provide unique identification